### ISSUE TITLE
fix: contrast ratios between chibivue theme colors and text color

### ISSUE THREAD

**Issue by yamanoku** [Ubugeeei/chibivue#244]
I am browsing in light mode and find the "Dive into book" link button on the TOP page to be more difficult to see than in dark mode.

| dark mode | light mode |
| ---- | ---- |
| ![Screenshot of the hero image section of the chibivue TOP page (in dark mode)](https://github.com/Ubugeeei/chibivue/assets/1996642/4c2bd56c-9a47-4471-8743-af28f5cba0bb) | ![Screenshot of the hero image section of the chibivue TOP page (in dark mode)](https://github.com/Ubugeeei/chibivue/assets/1996642/b0583ad2-4286-4586-813d-2275a0077fb0) |

I tried changing the text color to white in the developer tools and at first glance it seems to be easier to read, but the contrast ratio is lower than what is required by the accessibility requirements [^1]. More importantly, the black text has a slightly higher contrast ratio :-(

[^1]: Web Content Accessibility Guidelines - [Understanding Success Criterion 1.4.3: Contrast (Minimum) | WAI | W3C](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)

| text white | text black |
| ---- | ---- |
| ![Contrast ratio between theme color background and white color text is indicated as 2.92](https://github.com/Ubugeeei/chibivue/assets/1996642/8f12ceca-1d1d-4faa-b464-49c4161b35ad) | ![Contrast ratio between theme color background and black color text is indicated as 3.09](https://github.com/Ubugeeei/chibivue/assets/1996642/6b2f604b-081e-4da9-83f9-301c3ad1cb70) |

We would like to reconsider, as the brand color may be a difficult color to handle.

**Comment by Ubugeeei** [comment]
Thank you for being so courteous!!!!!!!!!!!!!!!!!!!!!!!!!!!!
